versioned_docs/version-12.x.x/release-notes/v12.3.6.md

```diff
@@ -0,0 +1,18 @@
+---
+id: v12.3.6
+title: Version 12.3.6 Release Notes
+sidebar_label: v12.3.6
+image: "img/release-note-link-preview.png"
+---
+
+_July 08th, 2024_
+
+## Console
+
+### Design Overview
+
+A fix has been addressed to allow the MongoDB change stream to subscribe to data changes even for large configurations (>6MB).

```

Comment:
@flower-of-the-bridges I kindly ask you a review on this content

Reply:
Hi @albertotessarotto sure, here it is!